2.train_model/README.md

```diff
@@ -0,0 +1,40 @@
+# 2. Train Model
+
+In this module, we train a ML model to predict phenotypic class from DeepProfiler-extracted features.
+
+We train the model in [train_model.ipynb](1.train_model.ipynb).
+We use [sklearn.linear_model.LogisticRegression](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html) for our machine learning model.
+We use the following parameters for our Logisic Regression model:
+
+- `penalty='elasticnet'`: Use elasticnet as the penalty for our model.
+Elastic-Net regularization is a combination of L1 and L2 regularization methods.
+The mixing of these two methods is determined by the `l1_ratio` parameter which we optimize later.
+- `solver='saga'`: We use the saga solver as this is the only solver that supports Elastic-Net regularization.
+- `max_iter=100`: Set the maximum number of iterations for solver to converge. 100 iterations allows the solver to maximize performance without completing unnecessary iterations.
+
+We use [sklearn.model_selection.GridSearchCV](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.GridSearchCV.html#sklearn.model_selection.GridSearchCV) to perform an exhaustive search for the parameters below. This searches for parameters that maximize the weighted F1 score of the model. We optimize weighted F1 score because this metric measures model precision and recall aand accounts for label imbalance (see [sklearn.metrics.f1_score](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html) for more details).
```

Comment:
```suggestion
We use [sklearn.model_selection.GridSearchCV](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.GridSearchCV.html#sklearn.model_selection.GridSearchCV) to perform an exhaustive search for the parameters below.
This searches for parameters that maximize the weighted F1 score of the model.
We optimize weighted F1 score because this metric measures model precision and recall aand accounts for label imbalance (see [sklearn.metrics.f1_score](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html) for more details).
```